resources/views/requests/show.blade.php

```diff
@@ -0,0 +1,129 @@
+@extends('layouts.master')
+
+@section('title', 'Report')
+
+@section('body')
+    <div class="row">
+        <div class="col">
+            <div class="card">
+                <div class="card-header">
+                    Request information
+                </div>
+
+                <div class="card-body">
+                    @if ($request->error_message)
+                        <div class="alert alert-danger" role="alert">
+                            <h6 class="alert-heading">Xebia Header Checker failed to execute the request</h6>
+
+                            {{$request->error_message}}
+                        </div>
+                    @endif
+
+                    <dl class="row">
+                        <dt class="col-sm-3">Url</dt>
+                        <dd class="col-sm-9">
+                            <a href="{{$request->endpoint->url}}" target="_blank">
```

Comment:
[Just in case (see blog post)!](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/) 😉